# 03 Term structure

Does the gap between five-print RWTC returns in backwardation and in contango keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: held.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'term-structure')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train mean five-print RWTC return in backwardation minus contango is -0.133%. Holdout difference is -0.055%. A negative gap means backwardation did not pay more than contango. The holdout kept the train sign.
Five-print RWTC log return by the sign of RCLC1 minus RCLC4
Train | Backwardation (spread > 0) | 3485 | -0.007%
Train | Contango (spread < 0) | 3949 | 0.126%
Holdout | Backwardation (spread > 0) | 717 | 0.196%
Holdout | Contango (spread < 0) | 349 | 0.251%


## Method

On each day both the spread and RWTC print, take the log change in RWTC five published prints later. Split the days by the sign of the spread.

## Result

Train mean five-print RWTC return in backwardation minus contango is -0.133%. Holdout difference is -0.055%. A negative gap means backwardation did not pay more than contango. The holdout kept the train sign.